hibernate-core/src/main/java/org/hibernate/annotations/BatchSize.java

```diff
@@ -19,19 +19,19 @@
  * collection.
  * <p>
  * For example:
- * <pre>{@code
- * @Entity
- * @BatchSize(size = 100)
- * class Product {
- *     ...
- * }
- * }</pre>
+ * <pre>
+ *    &#64;Entity
+ *    &#64;BatchSize(size = 100)
+ *    class Product {
+ *        ...
+ *    }
+ * </pre>
  * will initialize up to 100 lazy Product entity proxies at a time, but:
- * <pre>{@code
- * @OneToMany
- * @BatchSize(size = 5) /
- * Set<Product> getProducts() { ... };
- * }</pre>
+ * <pre>
+ *    &#64;OneToMany
+ *    &#64;BatchSize(size = 5) /
+ *    Set&lt;Product&gt; getProducts() { ... };
+ * </pre>
```

Comment:
we need to escape HTML special chars (including '<', '>' and '&'). See the current erroneous rending effect at https://docs.jboss.org/hibernate/orm/6.0/javadocs/org/hibernate/annotations/BatchSize.html